apps/portals/b2ai.standards/src/pages/OrganizationDetailsPage.tsx

```diff
@@ -77,184 +80,249 @@ function OrgHeaderCard({ id }) {
   )
 }
 
+export default function OrganizationDetailsPage() {
+  const { id } = useGetPortalComponentSearchParams()
+  const colExpressions = [
+    ORG_TABLE_COLUMN_CONSTS.MAIN_ORGANIZATION_JSON,
+    ORG_TABLE_COLUMN_CONSTS.ASSOCIATED_ORGANIZATION_JSON,
+    ORG_TABLE_COLUMN_CONSTS.RELEVANT_STANDARDS_JSON,
+    ORG_TABLE_COLUMN_CONSTS.GOVERNED_STANDARDS_JSON,
+    ORG_TABLE_COLUMN_CONSTS.DATASET_JSON,
+    ORG_TABLE_COLUMN_CONSTS.D4D,
+  ]
+  const queryBundleRequest: QueryBundleRequest =
+    getQueryBundleRequestWithIdFilter(
+      'Organization_denormalized',
+      colExpressions,
+      [id],
+    )
+  const {
+    data = [],
+    error,
+    isLoading,
+  } = useFetchJsonArrayLengths(queryBundleRequest)
+
+  if (error) {
+    return <ErrorBanner error={error} />
+  }
+
+  if (!id) {
+    return <ErrorPage type={SynapseErrorType.NOT_FOUND} gotoPlace={() => {}} />
+  }
+  if (isLoading) {
+    return <div>Loading...</div>
+  }
+  if (data.length !== 1) {
+    throw new Error(`Unexpected ${data.length} rows of Org data.`)
+  }
```

Comment:
Throwing an error here is not the best user experience. If there are 0, we should show a not found page. If there is more than 1, we can still show the first one.

```suggestion
  if (data.length == 0) {
    return <ErrorPage type={SynapseErrorType.NOT_FOUND} gotoPlace={() => {}} />
  } else if (data.length > 1) {
    console.error('Unexpected ${data.length} rows of Org data')
  }
```